### PR TITLE
fix: rework format for maxLength, minLength

### PR DESCRIPTION
**Pull request by vankop**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

#65 

### Breaking Changes

no

### Additional Info

Add support for `maxLength === 0`

### CODE REVIEW

File: test/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -1481,7 +1481,7 @@ exports[`Validation should fail validation for oneOf #1 1`] = `
 exports[`Validation should fail validation for oneOf #2 1`] = `
 "Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
  - configuration.optimization.runtimeChunk should be one of these:
-   boolean | \\"single\\" | \\"multiple\\" | object { name? }
+   boolean | \\"single\\" | \\"multiple\\" | empty string | object { name? }
```
Comment (evilebottnawi) [inline test/__snapshots__/index.test.js.snap]: `empty string`, regression?

Comment (vankop) [reply test/__snapshots__/index.test.js.snap]: what do you mean? it is by design, when `maxLength: 0`

Comment (vankop) [reply test/__snapshots__/index.test.js.snap]: Oh, I understood what you mean, I will check

Comment (vankop) [reply test/__snapshots__/index.test.js.snap]: @evilebottnawi I have updated `anyOf` in `configuration.optimization.runtimeChunk` with
```json
{
              "type": "string",
              "maxLength": 0
            },
```

Comment (evilebottnawi) [reply test/__snapshots__/index.test.js.snap]: :+1:

**Pull request merged**